internal/proto/replication.proto

```diff
@@ -206,4 +207,10 @@ message GetVolumeReplicationInfoResponse{
   // Holds the last sync time.
   // This field is REQUIRED.
   .google.protobuf.Timestamp last_sync_time = 1;
+  // This field is OPTIONAL.
+  .google.protobuf.Duration last_sync_duration = 2;
+  // This field is OPTIONAL.
+  // A value of 0 is equal to an unspecified field value.
+  // The value of this field MUST NOT be negative.
+  int64 last_sync_bytes = 3;
```

Comment:
Make sure to match the comments with csi-addons spec or close to it.

Reply:
This has to be edited in spec then, cause it's required to clear it here, will send another PR to update spec comments

Reply:
I meant to update it here.